packages/graphql/tests/schema/aggregations.test.ts

```diff
@@ -48,9 +48,7 @@ describe("Aggregations", () => {
               mutation: Mutation
             }
 
-            \\"\\"\\"
-            A BigInt value up to 64 bits in size, which can be a number or a string if used inline, or a string only if used as a variable. Always returned as a string.
-            \\"\\"\\"
+            \\"\\"\\"A BigInt value up to 64 bits in size, which can be a number or a string if used inline, or a string only if used as a variable. Always returned as a string.\\"\\"\\"
```

Comment:
It looks like this PR is reverting the change in #713 here:
https://github.com/neo4j/graphql/pull/713/files#diff-c0a37dcd426c9fb8df65d78d4f03fc92f83a8698885a5fd4f5b1b84cebbdfbad